src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyOptions.cs

```diff
@@ -27,6 +27,18 @@ public class CookiePolicyOptions
         /// </summary>
         public CookieSecurePolicy Secure { get; set; } = CookieSecurePolicy.None;
 
+        public CookieBuilder ConsentCookie { get; set; } = new CookieBuilder()
+        {
+            Name = ".AspNet.Consent",
```

Comment:
@blowdart  this cookie could use a better name, it doesn't say what is being consented to.